index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>판다마켓</title>
+</head>
+
+<body>
+    <header>
+        <a class="goto_root" href="./index.html">
+            <img src="./icons/Property 1=Variant3.svg" alt="main logo">
+        </a>
+        <a class="button-login" href="./login.html">
+            로그인
+        </a>
```

Comment:
```suggestion
        <a class="goto-root" href="./index.html">
            <img src="./icons/Property 1=Variant3.svg" alt="main logo">
        </a>
        <a class="button-login" href="./login.html">
            로그인
        </a>
```

선택자, 변수, 함수, 등등 식별자를 이름지어야 할 때 naming convention을 여러개 혼용해서 사용하지 않는게
코드를 읽기 쉽고, 이해하기 쉽게 만드는 방법중 하나 입니다.

naming convention은 종류가 많은데, 일반적으로 널리 알려진것은
camelCase, snake_case, kebaa-case, UPPERCASE,PascalCase 등이 있습니다.

예를들어, 자바스크립트는 대부분 camelCase를 사용하고, 종종 PascalCase를 사용하기도 합니다.

중요한것은 제품을 만드는 사람이 편한 것보다, 읽는 사람이 이해하기쉽게 볼 수 있게 만드는게 중요합니다.